openssh/README.md

```diff
@@ -0,0 +1,132 @@
+# openssh
+
+## Maintainers
+
+The Habitat Maintainers: <humans@habitat.sh>
+
+## Usage
+
+## Testing
+
+### Build the openssh core plan
+
+Enter a habitat studio
+
+```bash
+hab pkg build .
+```
+
+Exit habitat studio
+
+### Build and install the test plan
+
+Enter another habitat studio
+
+```bash
+set -x
+
+# set environment from the last openssh build
+source results/last_build.env
+
+# stop the habitat supervisor
+hab sup term
+
+# add some required software before building the openssh_test
+hab pkg install -b core/hab-studio
+hab pkg install -b core/busybox
+hab pkg install -b chef/inspec
+
+# install the openssh artifact
+hab pkg install -b results/$pkg_artifact
```

Comment:
Is it possible to not binlink? I think this may be the important point that @smacfarlane mentioned that we need to be super careful to test what we're building and not what's already available on the system.

Reply:
I've removed the bin link from inspec, but not from either core/busybox or the openssh installation.  I've added a note to the README.md regarding this